src/lib/run-dev.js

```diff
@@ -160,8 +162,11 @@ async function runDev (runOptions, config, _inprocHookRunner) {
   }
 
   const app = express()
+  app.use(express.text({ type: 'text/plain' }))
+  app.use(express.json({ strict: false }))
```

Comment:
Consider adding:

- `inflate: false`: express will decompress automatically but OpenWhisk does not so we should not give devs impression that just adding "Content-Type: gzip" will work.
- `limit: '1MB'`: to mimic OpenWhisk limitations

Reply:
thanks! will add those

Reply:
fixed in 76c4bc698d399481285846f55ec01bbfe2f72243